tests/v2/validation/provisioning/rke2/etcd_backup_restore.go

```diff
@@ -231,3 +231,21 @@ func watchAndWaitForPods(client *rancher.Client, clusterID string) error {
 	})
 	return err
 }
+
+func upgradeClusterK8sVersionWithUpgradeStrategy(client *rancher.Client, clustername string, k8sUpgradedVersion string, namespaceName string) error {
+	clusterObj, existingSteveAPIObj, err := getProvisioningClusterByName(client, clustername, namespaceName)
+	if err != nil {
+		return err
+	}
+
+	clusterObj.Spec.RKEConfig.UpgradeStrategy.ControlPlaneConcurrency = "10%"
```

Comment:
Shouldn't `ControlPlaneConcurrency` and `upgradeStrategy.WorkerConcurrency` be dynamic ?

Reply:
I made the values harcoded, because of the requirement in the [task](https://github.com/rancher/qa-tasks/issues/407#:~:text=Edit%20cluster%2C%20change%20the%20upgrade%20strategy%20to%20Control%20plane%20concurrency%20%2D%2010%25%2C%20worker%20nodes%20concurrency%3A%2020%25). The default values for ControlPlaneConcurrency and WorkerConcurrency is "10%"

Reply:
I see, thanks for the response @rishabhmsra . Please let me know if this is correct. 
If this configuration is not in the settings, internally the cluster will default to `10%` then.

In this case is this test only validating if `WorkerConcurrency` and `ControlPlaneConcurrency` exist and have these `10% and `20%` values ? 

Or are we interested in validating that the values of those settings too are not the default  ?  Like after upgrade validate if either of the values is different from the`10%` default.  (Using `15%` and `20%` for instance).

I understand the task use those specific values but I am trying to understand if those make sense for the test.

Reply:
The UI have these as default values if I am not mistaken. If not we can go with the default values.

Reply:
Yes Sowmya UI has the default value as 10%. 
@izaac, i got your point, I've updated the cp node concurrency to 15% and worker node concurrency is 20%.
So now the validation is something like this:
- With updated 15% and 20% concurrency values, cluster upgraded should be successful.
- once the cluster is upgraded, verify the concurrency values are 15% and 20%
- After the etcd restore validated both the values are at default 10%,